sdk/eventhub/Microsoft.Azure.EventHubs/tests/Processor/ProcessorNegativeCases.cs

```diff
@@ -96,7 +96,7 @@ public void InvalidPartitionManagerOptions()
             Assert.Throws<ArgumentException>(() =>
             {
                 TestUtility.Log("Setting lease duration outside of allowed range should fail.");
-                pmo.LeaseDuration = TimeSpan.FromSeconds(65);
+                pmo.LeaseDuration = TimeSpan.FromSeconds(1565);
```

Comment:
Would it make sense to not hard code the value but rather use MaxLeaseDurationInSeconds  + 65;

Reply:
MaxLeaseDurationInSeconds property is not accessible member of PartitionManagerOptions class.